### PR TITLE
add-new-kvm-health-check

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/4793

adding a new `k8s-kvm-health` image which also includes checks for kubelet and k8s-api healthz endpoints - https://github.com/giantswarm/k8s-kvm-health/pull/7

Also Increasing the timeouts for the health checks as it takes longer for a node to pull all images and start kubelet/api.

### CODE REVIEW

File: service/controller/v18/key/key.go [hunk 1]
```diff
@@ -29,16 +29,19 @@ const (
...
+	TimeoutSeconds = 5
 	// PeriodSeconds is PeriodSeconds param in liveness probe config
-	PeriodSeconds = 20
+	PeriodSeconds = 30
 	// FailureThreshold is FailureThreshold param in liveness probe config
 	FailureThreshold = 4
 	// SuccessThreshold is SuccessThreshold param in liveness probe config
 	SuccessThreshold = 1
 
+	// enable k8s-kvm-health check for k8s api
```
Comment (MarcelMue) [inline service/controller/v18/key/key.go]: Should be `Enable k8s-kvm-health check for k8s api.` not sure why we are so inconsistent in this file :man_shrugging:

File: service/controller/v18/version_bundle.go [hunk 2]
```diff
@@ -41,6 +36,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "kvm-operator",
-		Version: "3.1.1",
+		Version: "3.1.2",
```
Comment (MarcelMue) [inline service/controller/v18/version_bundle.go]: This number already changed in master afaik

**Review by MarcelMue — COMMENTED** [review]
I don't know much about how this works, code looks mostly fine though.

**Review by tuommaki — APPROVED** [review]
LGTM

**Review by r7vme — APPROVED** [review]

**Pull request merged**